Package.swift

```diff
@@ -213,6 +221,9 @@ let package = Package(
                 .headerSearchPath(".."),
                 .headerSearchPath("../include"),
                 .headerSearchPath("../../googletest/include"),
+            ],
+            linkerSettings: [
+                .linkedLibrary("swiftCore", .when(platforms: [.windows])), // for swift_addNewDSOImage
```

Comment:
Have to add this otherwise I get linker errors about "missing symbol swift_addNewDSOImage" -- @compnerd any chance you know why this is an issue on Windows (and WebAssembly) in particular?

Reply:
It is because you are using swiftc as the linker driver. This will add in the swift registrar (`swiftrt.obj` or `swiftrt.o`) which requires that you overlink against swiftCore. I think that we want to build sqlite statically and not as a DLL.

Reply:
Ah, got it. I think in the longer term we probably should change the build system to use swift as the linker driver only if there's any Swift sources built in the target, otherwise clang/clang++?

Reply:
Yes, that would make sense. We _could_ also use `-nostdlib` as a flag for the driver to exclude the registrar and the runtime.